master/usage/configuration/node.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Configuring a node IP address and subnet
+---
+
+Each Calico node has key routing configuration that may be specified in a number of
+different ways.  In particular, this includes the IPv4 and IPv6 addresses and 
```

Comment:
You're talking about every field in the calico `node` resource, right? What do you think about sticking a sample node yaml here?